app/Http/Controllers/ViewAssetsController.php

```diff
@@ -361,19 +377,23 @@ public function postAcceptAsset($logID = null)
                 ->update(array('assigned_to' => null));
             }
         }
-        $logaction->target_id = $findlog->target_id;
 
+        $logaction->target_id = $findlog->target_id;
         $logaction->note = e(Input::get('note'));
-        $logaction->user_id = $user->id;
```

Comment:
user_id has typically been a strictly admin id (it's how to populate "Admin" everywhere). There is no admin involved in the acceptance of a asset. It sort of doesn't make any sense to make them a target either, but everywhere else, user_id = admin actor